api/types.proto

```diff
@@ -359,6 +359,15 @@ message TaskStatus {
 	}
 }
 
+message AttachmentStatus {
```

Comment:
It is the status of an attachment. This is needed so that actors react on the notification based on the state, which would cause either an allocation or a release of the IP. It may also be needed in the dispatcher code to decide which attachment to send down to the agent, like only the allocated ones.